packages/rn-tester/js/examples/Accessibility/AccessibilityAndroidExample.android.js

```diff
@@ -5,30 +5,44 @@
  * LICENSE file in the root directory of this source tree.
  *
  * @format
+ * @flow strict-local
  */
 
 'use strict';
-
+const RNTesterBlock = require('../../components/RNTesterBlock');
```

Comment:
Nit: we can use `import ... from ...` here instead of using `const ... = require(...)`.